frontend/src/components/SelectQueryBuilder/SelectQueryBuilder.tsx

```diff
@@ -47,18 +47,6 @@ const SelectQueryBuilder: FunctionComponent<SelectQueryBuilderProps> = (props) =
     );
   }, [props.columns]);
 
-  const onSelectAll = () => {
```

Comment:
Revert all changes in here as well ... weren't asked for.

Reply:
Reverted